src/app/learnocaml_teacher_tab.ml

```diff
@@ -211,7 +211,7 @@ let rec teacher_tab token _select _params () =
             in
             let open_partition_ () =
               Lwt.async (fun () ->
-                ask_string ~title:"Choose a function name"
+                ask_string ~title:"Partitioning of student solutions"
```

Comment:
I am not sure that's a valid patch: we really want to ask a function name here.

Reply:
The question is below, this is only the title of the dialog. My reason for this change is that you get there by ctrl-click on an exercise (i.e. not an explicit button) and previously you may get pretty confused as to what this was about.
![image](https://github.com/ocaml-sf/learn-ocaml/assets/807966/7475afc3-564c-4692-b35a-adc46056d307)